From 9a732e96212e9cab96123d6883baf8963448bcd6 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Mon, 14 Oct 2019 23:36:34 +0900
Subject: [PATCH 1/1] cli: change default shared lcore in config

As primary is changed to do forwarding and require two lcores for,
change shared lcore among secondaries from 1 to 2.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 src/cli/config/default.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/cli/config/default.yml b/src/cli/config/default.yml
index 60aaef7..e8c7db5 100644
--- a/src/cli/config/default.yml
+++ b/src/cli/config/default.yml
@@ -11,7 +11,7 @@ sec_mem:
     val: -m 512
     desc: Mem size
 sec_m_lcore:
-    val: 1
+    val: 2
     desc: Shared master lcore among secondaries
 sec_wait_launch:
     val: 0.5
-- 
2.17.1

